tests/test_utils.py

```diff
@@ -271,27 +270,31 @@ class PlainRelatedResourceTypeSerializer(serializers.Serializer):
     assert get_related_resource_type(field) == output
 
 
-def test_get_included_serializers():
-    class IncludedSerializersModel(DJAModel):
-        self = models.ForeignKey("self", on_delete=models.CASCADE)
-        target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
-        other_target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
+class IncludedSerializersModel(DJAModel):
+    self = models.ForeignKey("self", on_delete=models.CASCADE)
+    target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
+    other_target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
 
-        class Meta:
-            app_label = "tests"
+    class Meta:
+        app_label = "tests"
 
-    class IncludedSerializersSerializer(serializers.ModelSerializer):
-        included_serializers = {
-            "self": "self",
-            "target": ManyToManyTargetSerializer,
-            "other_target": "tests.serializers.ManyToManyTargetSerializer",
-        }
 
-        class Meta:
-            model = IncludedSerializersModel
-            fields = ("self", "other_target", "target")
+class IncludedSerializersSerializer(serializers.ModelSerializer):
+    included_serializers = {
+        "self": "self",
+        "target": ManyToManyTargetSerializer,
+        "other_target": "tests.serializers.ManyToManyTargetSerializer",
+    }
+
+    class Meta:
+        model = IncludedSerializersModel
+        fields = ("self", "other_target", "target")
 
-    included_serializers = get_included_serializers(IncludedSerializersSerializer)
+
+def test_get_included_serializers():
```

Comment:
as this is not a utils anymore this test should actually be added to `test_serializers`. Unfortunantly those tests have not be converted yet but feel free to add it to `example/tests/test_serializers.py`